platform/broadcom/docker-syncd-brcm/bcmsh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+usage="$(basename "$0") [-h] [-q] -- interactive shell for bcm service
+
+where:
+    -h  show this help text
+    -q  quite, no banner (default: verbose)"
+
+banner="Press Enter to show prompt.
+Press Ctrl+C to exit.
+Run command 'exit' will shutdown bcm service.
```

Comment:
Based on my test,  exit/quit won't exit the service, this is good so we don't accidentally shut down syncd.  but we should remove this line of message

# /usr/bin/socat - UNIX-CONNECT:/var/run/sswsyncd/sswsyncd.socket

Enter 'quit' to exit the application.
drivshell>exit
exit

Failed to execute the diagnostic command. Error: Invalid parameter.
drivshell>quit
quit
Exiting the application.
Hit enter to get drivshell prompt..

Enter 'quit' to exit the application.
drivshell>